src/main/java/com/artipie/rpm/meta/XmlFile.java

```diff
@@ -58,8 +55,10 @@
  *  to XmlPackagesFile and renamed close by exploiting directly that information.
  *  Once it is done, use XmlPackagesFile in XmlFilelists, XmlOthers, XmlPrimary and keep
  *  using XmlFile in XmlRepomd.
+ * @checkstyle MethodsOrderCheck (500 lines)
  */
-final class XmlFile {
+@SuppressWarnings("PMD.TooManyMethods")
+final class XmlFile implements XmlWriter {
```

Comment:
@ammaratef45 we need to create an `envelope` for `XMLStreamWriter` interface. We can call it `XmlWriterWrap`. Then `XmlFile` class should extend `XmlWriterWrap` and provide additional `alterTag` method.